samples/azure/azure-functions/ASBFunctions_1/AzureFunctions.ASBTrigger/AzureServiceBusTriggerFunction.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.Azure.ServiceBus;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using NServiceBus;
+using System.Threading.Tasks;
+
+namespace AzureFunctions.ASBTrigger
+{
+    using NServiceBus.AzureFunctions.ServiceBus;
+
+    public class AzureServiceBusTriggerFunction
+    {
+        private const string EndpointName = "ASBTriggerQueue";
+        private const string ConnectionStringName = "ASBConnectionString";
+
+        [FunctionName(EndpointName)]
+        public static async Task Run(
+            [ServiceBusTrigger(queueName: EndpointName, Connection = "ASBConnectionString")]
```

Comment:
should that be:
```suggestion
            [ServiceBusTrigger(queueName: EndpointName, Connection = ConnectionStringName )]
```
?